cmds/init/init.go

```diff
@@ -137,24 +141,50 @@ func main() {
 	// run inito and then run our shell
 	// Perhaps we should stat inito first.
 	// inito is always first and we set default flags for it.
-	cloneFlags := uintptr(syscall.CLONE_NEWPID)
-	for _, v := range []string{"/inito", "/buildbin/uinit", "/buildbin/rush"} {
+	attr := &syscall.SysProcAttr{Cloneflags: uintptr(syscall.CLONE_NEWPID)}
+	if ! *test {
```

Comment:
You should look into build flags: https://golang.org/pkg/go/build/

From what I understand, you can use custom build flags, "test" for example.
Then pass `-tags=test` into the compiler to enable source files ending in `_test.go` or containing `// +build test`.

It will prevent extraneous test code from fattening up the build.